stubs/django-filter/django_filters/fields.pyi

```diff
@@ -0,0 +1,93 @@
+from typing import Any
+
+from django import forms
+
+from .conf import settings as settings
+from .constants import EMPTY_VALUES as EMPTY_VALUES
+from .utils import handle_timezone as handle_timezone
+from .widgets import (
+    BaseCSVWidget as BaseCSVWidget,
+    CSVWidget as CSVWidget,
+    DateRangeWidget as DateRangeWidget,
+    LookupChoiceWidget as LookupChoiceWidget,
+    RangeWidget as RangeWidget,
+)
+
+class RangeField(forms.MultiValueField):
+    widget: Any = ...
+    def __init__(self, fields: Any | None = ..., *args: Any, **kwargs: Any) -> None: ...
+    def compress(self, data_list: Any): ...
+
+class DateRangeField(RangeField):
+    widget: Any = ...
+    def __init__(self, *args: Any, **kwargs: Any) -> None: ...
+    def compress(self, data_list: Any): ...
+
+class DateTimeRangeField(RangeField):
+    widget: Any = ...
+    def __init__(self, *args: Any, **kwargs: Any) -> None: ...
+
+class IsoDateTimeRangeField(RangeField):
+    widget: Any = ...
+    def __init__(self, *args: Any, **kwargs: Any) -> None: ...
+
+class TimeRangeField(RangeField):
+    widget: Any = ...
```

Comment:
Should I clean up attribute assignments?

```suggestion
    widget: Any
```

Reply:
Fine to do this in a follow-up imo